src/fm_index.rs

```diff
@@ -120,9 +118,8 @@ where
     ///
     /// Sampled suffix array data is stored in this index.
     pub fn size(&self) -> usize {
```

Comment:
Maybe the name of this method `size` is a bit vague - should we name it `heap_size` like vers-vec?

Reply:
I'd be happy to see this change!

Reply:
Filed #58 not to forget.